scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -227,7 +230,8 @@ function Get-SecurityDaemon {
             Invoke-WebRequest `
                 -Uri "https://aka.ms/iotedged-windows-latest" `
                 -OutFile "$ArchivePath" `
-                -UseBasicParsing
+                -UseBasicParsing `
+                -Proxy $Proxy
```

Comment:
What happens if `$Proxy` isn't set?

Reply:
Yeah, I made sure that if $Proxy is not set, it's the same as if you didn't pass the -Proxy argument to `Invoke-WebRequest` at all.